### ISSUE TITLE
how to send and receive messages in the meeting

### ISSUE THREAD

**Issue by jade0520** [microsoftgraph/microsoft-graph-comms-samples#791]
Is there a way for the bot to send and receive messages in the chat window within a Teams meeting while it has joined the meeting?

**Comment by InDieTasten** [comment]
Yes, but chat functionality is completely separated from the calling functionality in the SDKs. You can take a look here: https://learn.microsoft.com/en-us/graph/api/resources/chat?view=graph-rest-1.0

If you are connected to a meeting, you should be able to get the chatInfo resource to perform the chat operations using the regular Graph SDK  for example.

**Comment by yk-recall** [comment]
Confirming that InDieTasten is correct here, you’ll want to leverage the chat resource type to interact with the chat window in a Teams meeting. 

You also might want to check out the Recall.ai API.

It’s a simple 3rd party API that lets you use meeting bots to get raw audio/video/metadata from meetings without you needing to spend months to build, scale and maintain these bots.

Here are the API docs: https://docs.recall.ai

**Comment by InDieTasten** [comment]
@yk-recall I would like to remind you that we have a https://opensource.microsoft.com/codeofconduct/ here. Your various comments in this repo are adding no value to the issues at hand. It is quite clear, that you copy some detail from the original issue and claim that recall.ai solves it, giving no details whatsoever.
I would appreciate, if you could tone it down, or limit your interactions to issues, where you can add proper info to the discussion.
The CoC does not prohibit advertising, but the way you do it you just take your space to advertise and give nothing of value to the community. It's disrespectful.
